cfgov/jinja2/v1/_includes/molecules/related-metadata.html

```diff
@@ -0,0 +1,109 @@
+{# ==========================================================================
+
+   related_metadata.render()
+
+   ==========================================================================
+
+   Description:
+
+   Creates related metadata markup when given:
+
+   is_half_width:         A Boolean indicating whether the list items should be at half
+                          width. Defaults to False.
+
+   related_metadata_list: A list of relatated metadata items.
+
+   ========================================================================== #}
+
+{% macro render(related_metadata_list, is_half_width=false) %}
```

Comment:
Should this be `related_metadata` like in https://github.com/cfpb/cfgov-refresh/pull/1323#discussion_r49383746